hvsock.go

```diff
@@ -12,15 +13,73 @@ import (
 	"time"
 	"unsafe"
 
+	"golang.org/x/sys/windows"
+
 	"github.com/Microsoft/go-winio/pkg/guid"
+	"github.com/Microsoft/go-winio/pkg/sockets"
 )
 
-//sys bind(s syscall.Handle, name unsafe.Pointer, namelen int32) (err error) [failretval==socketError] = ws2_32.bind
+const afHvSock = 34 // AF_HYPERV
+
+var (
+	// https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/make-integration-service#vmid-wildcards
+
+	// HVguidWildcard is the wildcard VmId for accepting connections from all partitions.
+	HVguidWildcard = guid.GUID{} // 00000000-0000-0000-0000-000000000000
```

Comment:
I would suggest sticking with the current `Hvsock` prefix used in winio, and Go idioms, and name these like `HvsockGUIDWildcard`, `HvsockGUIDBroadcast`, etc.

Reply:
The last word in the name should still be capitalized (`Wildcard`, `Loopback`, etc.).